### ISSUE TITLE
Liquid fire adapter?

### ISSUE THREAD

**Issue by thedig** [acoustep/ember-cli-foundation-6-sass#15]
Hi there,

Curious how high of priority the liquid fire adapter is?

Best,
.dig

**Comment by acoustep** [comment]
I believe this is on @techiemac's todo list as he has been doing the majority of the Javascript for this package. 

It's really up to when he gets time or if someone is willing to do a PR for it.

**Comment by techiemac** [comment]
Yes, this is on my todo list but I have been swamped with work for the past several weeks.

I hope to take a look at this in the coming weeks.
